### PR TITLE
Prepare release 1.2

### PR DESCRIPTION
**Pull request by dmoranj**

### CODE REVIEW

**Comment by fgalan** [comment]
This is the "sister PR" in perseo-fe: https://github.com/telefonicaid/perseo-fe/pull/194, just in case you can to see an compare.

File: scripts/build/release.sh [hunk 1]
```diff
@@ -40,7 +40,7 @@ Usage:
         The second argument indicates what type of release is it going to be released:
 
         - sprint: releases that are meant to be created each sprint end. A tag is automatically
-        generated along with the branch and develop is merged with master.
+        generated along with the branch.
```
Comment (fgalan) [inline scripts/build/release.sh]: Note L47 below... "master is not updated" is not strictly correct... Have a look to https://github.com/telefonicaid/perseo-fe/pull/194/files#diff-db01dd9c2e5c291a004d2e3d3c73bff6L47

File: scripts/build/release.sh [hunk 2]
```diff
@@ -181,21 +181,20 @@ touch $CHANGELOG_FILE
...
     git commit -m "ADD Step: $currentVersion -> $NEW_VERSION"
-    git push origin develop
+    git push origin master
 
     # We do the tag only and merge to master only in the case of  non "dev" release
     if [ "$PEP_RELEASE" = "sprint" ]
     then
        git checkout master
        git pull origin master
-       git merge develop
```
Comment (fgalan) [inline scripts/build/release.sh]: You are already in master and you have just pushed master to origin, so lines 196 and 197 could be also removed (see https://github.com/telefonicaid/perseo-fe/pull/194/files#diff-db01dd9c2e5c291a004d2e3d3c73bff6L201)

**Comment by dmoranj** [comment]
I have updated the release script, to remove some redundant lines.

File: scripts/build/release.sh (hunk 1 above)
Comment (dmoranj) [inline scripts/build/release.sh]: Fixed in 	3b48c15

**Comment by fgalan** [comment]
LGTM

**Pull request merged**